tiatoolbox/tools/registration/wsi_registration.py

```diff
@@ -154,6 +156,9 @@ def prealignment(
     )
     dice_before = dice(padded_fixed_mask, padded_moving_mask)
 
+    cy, cx = ndi.center_of_mass((1 - fixed_img) * fixed_mask)
```

Comment:
Please check this if it needs to be removed.

Reply:
removed